fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/RegistrySyncManager.java

```diff
@@ -130,13 +133,44 @@ public static CompoundTag toTag(boolean isClientSync) {
 				}
 			}
 
-			if (REGISTRY_BLACKLIST.contains(registryId)) {
+			CompoundTag existingRegistryData = null;
```

Comment:
It is quite unclear from a quick look what "existingRegistryData" and "activeIdMap" are supposed to mean

Reply:
I will go around and put some comments on things.